Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -89,23 +94,36 @@ public struct Configuration {
     }
 
     public init?(yaml: String) {
-        let yamlResult = Yaml.load(yaml)
```

Comment:
Was this done intentionally? I think this incorrectly reverts the change introduced in a1f734e, which prints more informative errors when encountering misconfigured YAML.